src/main/java/net/mcreator/ui/modgui/BiomeGUI.java

```diff
@@ -605,16 +605,25 @@ private void updateBiomeTreesForm() {
 		biome.name = name.getText();
 		biome.groundBlock = groundBlock.getBlock();
 		biome.undergroundBlock = undergroundBlock.getBlock();
-		if (customTrees.isSelected())
-			biome.treeType = biome.TREES_CUSTOM;
-		else
-			biome.treeType = biome.TREES_VANILLA;
 		biome.airColor = airColor.getColor();
 		biome.grassColor = grassColor.getColor();
 		biome.foliageColor = foliageColor.getColor();
 		biome.waterColor = waterColor.getColor();
 		biome.waterFogColor = waterFogColor.getColor();
-		biome.treesPerChunk = (int) treesPerChunk.getValue();
+
+		if(customTrees.isSelected()){

```

Comment:
Pass selected values in either case (even if custom trees is not checked), so UI settings are saved.